src/tests/ftest/osa/osa_offline_parallel_test.py

```diff
@@ -143,17 +155,40 @@ def run_offline_parallel_test(self, num_pool, data=False):
             self.assertTrue(pver_end == 25,
                             "Pool Version Error:  at the end")
         if data:
-            self.verify_single_object()
+            self.run_ior_thread("Read", oclass, test_seq)
+            self.run_mdtest_thread()
+            self.container = self.pool_cont_dict[self.pool][0]
+            kwargs = {"pool": self.pool.uuid,
+                      "cont": self.container.uuid}
+            output = self.daos_command.container_check(**kwargs)
```

Comment:
(pylint-no-member) Instance of 'OSAOfflineParallelTest' has no 'daos_command' member